### DISCUSSION TITLE
Maintainers

### DISCUSSION THREAD

**Discussion by BenKnigge** [go-sqlx/sqlx#13] [Ideas]
I would like it if this fork had multiple maintainers and a policy regarding how something gets merged. 

I am proposing that there will be 3 - 5 maintainers and a simple majority will need to agree before a pr is merged. 

If a maintainer is unable or unwilling to participate for an extended period of time they should be replaced. 

This should mitigate the possibility of bad code getting merged and prevent the project from becoming abandoned again in the future.

**Comment by uvulpos** [comment]
Sounds good to me, with just one addition: we have a team members chat. It could be something free and open to join, like e.g., slack (like I know LF Energy uses that), or discord, mattermost,... or whatever you prefer

**Reply by BenKnigge** [reply]
I've seen this one used before https://gitter.im/ slack is supper popular and mattermost is another go project I would prefer one of those three.

**Reply by uvulpos** [reply]
Not really into matrix stuff, if you want to take care of it, go ahead :) Besides that, I have no preferences really. It should be (in the best case) simple, no extra coasts to host and welcome new people. Rocketchat is also another opportunity.

**Reply by uvulpos** [reply]
For now, I would like to create a discord community because I think it is most accessible to most users. If me feel the need, we could switch later. What do you think? https://discord.gg/cvuQRPx76y

**Reply by BenKnigge** [reply]
Sounds good to me.
